packages/clay-drop-down/README.mdx

```diff
@@ -162,3 +162,4 @@ const menus = {
 ## Caveats
 
 One caveat with the drop down menu is that it is rendered inside of a [React Portal](https://reactjs.org/docs/portals.html) and is rendered directly to the `body` element. This means if you are using the `menuWidth` prop set to `auto`, it will not respect the size of the node parent of the drop down, since the menu is rendered directly to the body.
+Another important thing to have in mind, is that `<DropDownWithDrilldown />`, will render it's menus in the order that they are specified. This means that if you specify the menus in the wrong order, the menu animation will not behave correctly.
```

Comment:
I've been thinking about adding this to the DropDownWithDrilldown section as an alert to make it more visible to people, what do you think?

Something like that:

```html
<div class="clay-site-alert alert alert-warning">
    Another important thing to have in mind, is that <code class="language-text"><DropDownWithDrilldown /></code>, will render it's menus in the order that they are specified. This means that if you specify the menus in the wrong order, the menu animation will not behave correctly.
</div>
```

Reply:
Hey @matuzalemsteles, I had the idea of using an "alert" but without the code example because it renders something like this and (for me) it's not clear at all 

![2021-06-09-11:34:03](https://user-images.githubusercontent.com/5572/121330886-afc02200-c916-11eb-8403-2722f2d06985.png)

I read this as "Another important thing to have in mind is that "Click me" ....", ... what is "Click me" ? On top of that to see what we're talking about we're going to have to click on the button to expand the code snippet.

I think something like this would be easier to understand, but tell me what you think when you can 

![2021-06-09-11:39:41](https://user-images.githubusercontent.com/5572/121331686-67edca80-c917-11eb-9bbe-a563c047888f.png)

Reply:
Oh no, sorry this will render the example component, I forgot about the mdx in action here when adding `<DropDownWithDrilldown />` because this is imported above, I think the second image is enough for us to get on with it.

Reply:
No problem, I thought it was a mistake too. I updated that in [761121e](https://github.com/liferay/clay/commit/761121ee59510731c34e252b1c958269695742d1)